sci-electronics/systemc/systemc-2.3.4-r1.ebuild

```diff
@@ -41,12 +40,30 @@ src_configure() {
 		--with-unix-layout
 }
 
+src_install() {
+	default
```

Comment:
should `default` go after `DOCS=`?

Reply:
> should `default` go after `DOCS=`?

Thanks for review, resolved, after fully test, I found the files which in ``DOCS=`` will installed by ``Makefile``, so this ebuild don't need set ``DOCS``, I removed it in https://github.com/gentoo/gentoo/pull/19033/commits/a6489014eabdc12498ae169a6b7feb03230cb9f3